src/components/Table/Table.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { FC, ReactNode } from 'react';
+import classNames from 'classnames';
+import Spinner from '../Spinner/Spinner';
+import styles from './Table.module.scss';
+
+interface TableProps {
+  /**
+   * Additional classes to add.
+   */
+  className?: string;
+  /**
+   * The children components to render inside the table
+   */
+  children?: ReactNode;
+  /**
+   * Enable a hover state on table rows.
+   */
+  hoverableRows?: boolean;
+  /**
+   * Remove borders around table, thead, tbody, td, etc.
+   */
+  isBorderless?: boolean;
+   /**
+   * Make Table more compact by cutting cell padding in half.
+   */
+  isCompact?: boolean;
+  /**
+   * Responsive tables allow tables to be scrolled horizontally with ease.
+   * If table overruns horizontal width of container, allow for horizontal scrolling.
+   */
+  isResponsive?: boolean;
+  /**
+   * Adds zebra-striping to any table row within the table body.
+   */
+  isStriped?: boolean;
+  /**
+   * Set to true if data is loading.
+   */
+  isLoading?: boolean;
+  /**
+   * Fix the width of the columns. Can be useful if sorting is enabled and the content of
+   * the columns is changing; prevents the horizontal jump when this occurres.
+   */
+  useFixedWidthColumns?: boolean;
+}
+
+const Table: FC<TableProps> = ({
+  className = undefined,
```

Comment:
There's nothing required to render a Table; this seems odd.

Reply:
@JCarmine I get the concern here, but not sure it's necessarily a dealbreaker, after all anybody could render a `<table>` with no content, or a `<button>` or a `<p>`. Any components that accept children, like semantic html elements, will require the user to write some basic markup in order to actually render content. We have examples of these with our `Button` component as well. An empty button will render an empty button.